README.md

```diff
@@ -1 +1,9 @@
-# hcommon-hive-metastore
\ No newline at end of file
+# hcommon-hive-metastore
+
+# Start using
+You can obtain hcommon-hive-metastore from Maven Central :
+
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.hotels/hcommon-hive-metastore/badge.svg?subject=com.hotels:hcommon-hive-metastore)](https://maven-badges.herokuapp.com/maven-central/com.hotels/hcommon-hive-metastore) [![Build Status](https://travis-ci.org/HotelsDotCom/hcommon-hive-metastore.svg?branch=master)](https://travis-ci.org/HotelsDotCom/hcommon-hive-metastore) [![Coverage Status](https://coveralls.io/repos/github/HotelsDotCom/hcommon-hive-metastore/badge.svg?branch=master)](https://coveralls.io/github/HotelsDotCom/hcommon-hive-metastore) ![GitHub license](https://img.shields.io/github/license/HotelsDotCom/hcommon-hive-metastore.svg)
+
+# Overview
+General purpose libraries for interacting with the Hive metastore
```

Comment:
Please add a "Legal" section at the bottom - you can take it from any of our other OS projects, just update the dates accordingly.